.github/workflows/maven.yml

```diff
@@ -31,6 +31,6 @@ jobs:
       uses: actions/upload-artifact@v2
       with:
         name: "JPlag"
-        path: "jplag/target/jplag-*-jar-with-dependencies.jar"
+        path: "jplag/target/jplag-*-jar-with-deps-merge-services.jar"
```

Comment:
fixed using shade instead of the assembly plugin.

As a side effect, it is no longer necessary to call specifically `assembly:single` in order to have the uber jar, this is already included in the `package` phase